src/utils/ping.ts

```diff
@@ -0,0 +1,9 @@
+import fetch from 'node-fetch';
+
+class PingClass {
```

Comment:
Are your planning to add anything to this class? If not I guess it makes sense to not abstract a single `fetch` call